### DISCUSSION TITLE
JSON file is not fully checked

### DISCUSSION THREAD

**Discussion by GustavoDcs** [super-linter/super-linter#5804] [Q&A] [answered]
Hey everyone, using version 6.6.0.

After scanning a JSON file, if a problem is found, the linting process stops.

Is this the expected behavior? For example, I had a comment on line 10 and another on line 15. On the first run, it detected the comment on line 10 and terminated with exit code 1, despite additional errors below.

Is there a way to validate and show all of the problems despite finding the first one?

```
  2024-06-26 06:19:37 [INFO]   Linting JSON items...
  Error: -26 06:19:42 [ERROR]   Found errors when linting JSON. Exit code: 1.
  2024-06-26 06:19:42 [INFO]   Command output for JSON:
  ------
  /github/workspace/tsconfig.json
    34:30  error  Parsing error: Unexpected token ','
  ✖ 1 problem (1 error, 0 warnings)
```

**Comment by ferrarimarco** [comment, marked as answer]
Hi @GustavoDcs !

This likely depends on ESlint and eslint-plug-json (the linter and plugin we use to lint JSON files). Super-linter just invokes them.

I would check in their repositoritories and documentation to get the answer to this question.
